yt/frontends/boxlib/data_structures.py

```diff
@@ -1405,6 +1403,16 @@ def _read_header(raw_file, field):
                 lo_corner, hi_corner, node_type = _line_to_numpy_arrays(clean_line)
                 boxes.append((lo_corner, hi_corner, node_type))
 
+            try:
+                # nghost_line is a single number
+                ng = int(nghost_line[0])
+                ndims = len(lo_corner)
+                nghost = np.array(ndims * [ng])
+            except ValueError:
+                # nghost_line is (#,#,#)
```

Comment:
This comment is helpful _but_ I think it's slightly misleading because you are specifying the format of `nghost_line[0]`, not `nghost_line` (and I have no idea what the rest of the list is... is the `.split()` a security to exclude possible comments ?)